.gitignore

```diff
@@ -2,6 +2,7 @@
 /auth0
 /test/integration/identifiers
 coverage*
+/dist
```

Comment:
Moreso curious about this directory, is it ignoring a current set of release artifacts or is this for future work?

Reply:
This directory is an artifact of running GoRelease. Since I'm doing that in development to test the config (using snapshots) I'm ignoring the artifacts so that they don't accidentally get into the repo.